assets/js/components/notifications/AdSenseAlerts.js

```diff
@@ -39,60 +39,56 @@ function AdSenseAlerts() {
 	const accountID = useSelect( ( select ) =>
 		select( MODULES_ADSENSE ).getAccountID()
 	);
+
+	const [ registeredNotifications, setRegisteredNotifications ] = useState(
+		[]
+	);
+	const { registerNotification } = useDispatch( CORE_NOTIFICATIONS );
+
 	const notifications = useSelect( ( select ) =>
 		select( MODULES_ADSENSE ).getNotifications()
 	);
 
-	if (
-		! adSenseModuleConnected ||
-		! accountID ||
-		notifications === undefined
-	) {
-		return null;
-	}
+	useEffect( () => {
+		// If AdSense is not connected, or the account ID is not set,
+		// there's nothing to do here.
+		if ( ! adSenseModuleConnected || ! accountID ) {
+			return;
+		}
 
-	return (
-		<Fragment>
-			{ notifications.map(
-				( {
-					id,
-					title,
-					description,
-					format,
-					ctaURL,
-					ctaLabel,
-					ctaTarget,
-					severity,
-					isDismissable,
-				} ) => (
-					<BannerNotification
-						className="googlesitekit-adsense-alert"
-						key={ id }
-						id={ id }
-						title={ title || '' }
-						description={ description }
-						WinImageSVG={ NotificationAlertSVG }
-						format={ format || 'small' }
-						ctaLink={ ctaURL }
-						ctaLabel={ ctaLabel }
-						ctaTarget={ ctaTarget }
-						type={ severity }
-						dismiss={ __( 'OK, Got it!', 'google-site-kit' ) }
-						isDismissible={ isDismissable || true }
-						module="adsense"
-						moduleName={ _x(
-							'AdSense',
-							'Service name',
-							'google-site-kit'
-						) }
-						dismissExpires={ 0 }
-						showOnce={ false }
-						logo
-					/>
-				)
-			) }
-		</Fragment>
-	);
+		// Register any notifications from the server that haven't yet been
+		// registered.
+		//
+		// (Usually there will be one, if any, notification from the server.)
+		notifications?.forEach( ( notification ) => {
+			if ( registeredNotifications.includes( notification.id ) ) {
+				return;
+			}
+
+			registerNotification( notification.id, {
+				Component() {
+					return <NotificationFromServer { ...notification } />;
+				},
+				priority: notification.priority,
+				areaSlug: NOTIFICATION_AREAS.DASHBOARD_TOP,
+				isDismissible: notification.isDismissible || true,
```

Comment:
I see this is was how it was before, but that means that these would always be dismissible 🤔 

Looking at the backend, these are hard-coded to `true`, so we should do the same here or pass the value directly without a fallback.